src/components/organisms/settings/organization/users.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+
+import { useTranslation } from "react-i18next";
+
+import { ModalName } from "@src/enums/components";
+import { useModalStore } from "@src/store";
+
+import { Button, IconButton, TBody, THead, Table, Td, Th, Tr, Typography } from "@components/atoms";
+import {
+	DeleteUserFromOrganizationModal,
+	OrganizationUserCreateModal,
+} from "@components/organisms/settings/organization";
+
+import { RotateRightIcon, TrashIcon } from "@assets/image/icons";
+
+export const OrganizationUsers = () => {
+	const { t } = useTranslation("settings", { keyPrefix: "organization.users" });
+	const { openModal } = useModalStore();
+
+	return (
+		<div className="w-3/4">
+			<Typography className="mb-4 font-bold" element="h2" size="xl">
+				{t("title")}
+			</Typography>
+			<Button
+				className="ml-auto border-black bg-white px-5 text-base font-medium hover:bg-gray-950 hover:text-white"
+				onClick={() => openModal(ModalName.organizationUserCreate)}
+				variant="outline"
+			>
+				{t("buttons.addUser")}
+			</Button>
+			<Table className="mt-6">
+				<THead>
+					<Tr>
+						<Th className="w-1/4 pl-4 font-normal">{t("table.headers.name")}</Th>
+						<Th className="w-1/4 pl-4 font-normal">{t("table.headers.email")}</Th>
+						<Th className="w-1/4 pl-4 font-normal">{t("table.headers.status")}</Th>
+						<Th className="w-1/4 pl-4 font-normal">{t("table.headers.role")}</Th>
```

Comment:
We don't need `pl-4` for all `Th` and `Td`, only the first one.
We don't need `font-normal` because the `Th` is `font-normal` by default.

Each cell can't be 1/4, because we have 5 cells, which means it should be 1/5 at least.
Also, try to make it relative to the content.
For example - Actions is the smallest width, so it can be 1/8, Role is also short, it also can be 1/8, others can be 1/4. Try to play with it to make it proportional.

Also please add `min-w-X` to each header/cell.

Reply:
done

Reply:
I don't see you added `min-w-X`, all the rest looks great

Reply:
yes, added*